spec/features/idv/analytics_spec.rb

```diff
@@ -34,6 +34,7 @@
       'IdV: phone of record visited' => {},
       'IdV: phone confirmation form' => { success: true, errors: {}, phone_type: :mobile, types: [:fixed_or_mobile], carrier: 'Test Mobile Carrier', country_code: 'US', area_code: '202' },
       'IdV: phone confirmation vendor' => { success: true, errors: {}, vendor: { messages: [], exception: nil, context: { stages: [{ address: 'AddressMock' }] }, transaction_id: 'address-mock-transaction-id-123', timed_out: false }, new_phone_added: false },
+      'IdV: final resolution' => { success: true },
```

Comment:
We shouldn't need to merge the entry below in `FSMv1` anymore, should we?

Reply:
Yup - fixed in 1550bec